src/rsht.c

```diff
@@ -10,10 +10,13 @@
  * WITHOUT ANY WARRANTY. See the LICENSE file for more details.
  */
 
+
+#ifndef RSHT_NO_INCLUDES
```

Comment:
There's already another pull request open for this: https://github.com/robstolarz/rsht/pull/16

Do you mind splitting this change out of this PR?